### ISSUE TITLE
Installation instructions incorrectly reference location of .env

### ISSUE THREAD

**Issue by catherdersoftware** [ZOSOpenTools/meta#660]
The instructions say that the .env file is in the `bin` subdirectory of `meta`. It is not, it is in root.

**Comment by IgorTodorovskiIBM** [comment]
Good catch, it is fixed now: https://github.com/ZOSOpenTools/meta/blob/main/README.md

**Comment by MikeFultonDev** [comment]
@catherdersoftware please close off if you are ok with the fix
